desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTabContents/StorageBrowserTabContent.scss

```diff
@@ -39,4 +39,16 @@
       font-weight: vars.$fluidx-heading-h3-weight;
     }
   }
+
+  .hue-storage-browser__path-browser-panel {
+    display: flex;
+    margin-top: vars.$fluidx-spacing-xxs;
+
+    .hue-storage-browser__filePath {
```

Comment:
we are repeating `.hue-storage-browser` in every class name. even though it is nested. we can just name the class as filePath

Reply:
I've named it in accordance to BEM

Reply:
Do we need to nest these classes in `.hue-storage-browser-tabContent`?